ds_judgements_public_ui/templates/layouts/static_content.html

```diff
@@ -1,4 +1,8 @@
 {% extends "layouts/base.html" %}
+{% block meta_description %}
+  <meta name="description"
```

Comment:
It would be better if this entire meta tag was wrapped in the `if` statement, so we don't render it at all if the description is missing. At the moment it will say the description is just an empty string.

Reply:
updated wrapped it @jacksonj04